networkx/classes/multidigraph.py

```diff
@@ -277,6 +278,19 @@ def __init__(self, incoming_graph_data=None, **attr):
             packages are installed the data can also be a NumPy matrix
             or 2d ndarray, a SciPy sparse matrix, or a PyGraphviz graph.
 
+        multigraph_input : bool or None (default None)
+           Note: Only used when `incoming_graph_data` is a dict.
+           If True, `incoming_graph_data` is assumed to be a
+           dict-of-dict-of-dict-of-dict structure keyed by
+           node to neighbor to edge keys to edge data for multi-edges.
+           A NetworkXError is raised if this is not the case.
+           If False, :func:`to_networkx_graph` is used to try to determine
+           the dict's graph data structure as either a dict-of-dict-of-dict
+           keyed by node to neighbor to edge data, or a dict-of-iterable
+           keyed by node to neighbors.
+           If None, the treatment for True is tried, but if it fails,
+           the treatment for False is tried.
+
```

Comment:
It looks like the [rendered docstrings](https://167-2055378-gh.circle-artifacts.com/0/doc/build/html/reference/classes/multigraph.html) are being built from the class docstring rather than the `__init__` docstring, so I think this should be copied up to the `Parameters` section in the class docstring as well. Same goes for the DiGraph

Reply:
Thank you!!  I missed that completely.  Looks like I mis-indented the description too, so I fixed that.
If you get a chance to see whether this did what you suggested, great -- if not, enjoy your gatherings! and good luck with your trip!

Reply:
circleci still seems to be having problems, but it looks good now on a local build! :+1: